config.json

```diff
@@ -21,6 +22,10 @@
             "error"
         ]
     },
+		"cache": {
```

Comment:
Can you move this below API? Or even inside API since caching is related to API?